app/src/vuex/modules/user.js

```diff
@@ -0,0 +1,72 @@
+import config from "src/../config"
+import firebase from "../../firebase"
+import * as Sentry from "@sentry/browser"
+
+const Auth = firebase.auth()
+
+export default () => {
+  const state = {
+    userSignedIn: false,
+    user: null,
+    signInError: null,
+    externals: {
+      config,
+    },
+  }
+
+  const mutations = {
+    userSignedIn(state, hasSignedIn) {
+      state.userSignedIn = hasSignedIn
+    },
+    setUserInformation(state, user) {
+      state.user = user
+    },
+    setSignInError(state, error) {
+      state.signInError = error
+    },
+  }
+
+  const actions = {
+    signInUser({ commit }, { user }) {
+      commit(`userSignedIn`, true)
+      commit(`setUserInformation`, user)
+      console.log("User is now signed in!")
+    },
+    sendUserMagicLink({ commit }, { user }) {
+      const actionCodeSettings = {
+        url: `http://localhost:9080/email-authentication`,
```

Comment:
I know, but how was I supposed to try it otherwise? I mean, I needed to code the component.

Actually, I was thinking, isn't it more comfortable to log in via password? We could give the two options no? All sites I know work with password. And email is used to reset password.

I can look into password reset with firebase. It is all very straight forward and well documented, all this authentication thing

Reply:
We want a simple system. Slack i.e. works only with magic links

Reply:
Can't you get the current base url with js to insert here together with the callback route?

Reply:
oh, ok, good idea!

Reply:
Done! :woman_dancing: